mlflow/transformers/__init__.py

```diff
@@ -744,12 +818,33 @@ def log_model(
 
     Args:
         transformers_model:
-            A trained transformers `Pipeline` or a dictionary that maps required components of a
-            pipeline to the named keys of ["model", "image_processor", "tokenizer",
-            "feature_extractor"]. The `model` key in the dictionary must map to a value that
-            inherits from `PreTrainedModel`, `TFPreTrainedModel`, or `FlaxPreTrainedModel`.
-            All other component entries in the dictionary must support the defined task type that is
-            associated with the base model type configuration.
+            The transformers model to save. This can be one of the following format:
+
+                1. A transformers `Pipeline` instance.
+                2. A dictionary that maps required components of a pipeline to the named keys
+                    of ["model", "image_processor", "tokenizer", "feature_extractor"].
+                    The `model` key in the dictionary must map to a value that inherits from
+                    `PreTrainedModel`, `TFPreTrainedModel`, or `FlaxPreTrainedModel`.
+                    All other component entries in the dictionary must support the defined task
+                    type that is associated with the base model type configuration.
+                3. A string that represents a path to a local/DBFS directory containing a model
+                    checkpoint. The directory must contain a `config.json` file that is required
+                    for loading the transformers model. This is particularly useful when logging
+                    a model that cannot be loaded into memory for serialization.
+
+            An example of submitting a `Pipeline` from a default pipeline instantiation:
```

Comment:
```suggestion
            An example of logging a `Pipeline` from a default pipeline instantiation:
```

can we use logging instead of submitting?